src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/WebController.java

```diff
@@ -29,6 +29,8 @@ public String getLoginPage(Model model, OAuth2AuthenticationToken oAuth2Authenti
         Map<String, String> urls = new HashMap<>();
 
         // get around an unfortunate limitation of the API
+	// NOTE: one of the two links created using this data is the login with Github link on the /login page. We want to remove this link,
+	// so we should try to remove that from the database later on
```

Comment:
This is still showing up because the `application.properties` file is still trying to create a GitHub OAuth Client due to the presence of GitHub scopes. Removing the scope line, as well as the GitHub Org line, should fix the issue.

Once you fix this, remove this comment in the code.